kf_lib_data_ingest/common/constants.py

```diff
@@ -15,6 +15,15 @@ class COMMON:
     MULTIPLE = "Multiple"
 
 
+class FILE:
+    class HASH:
+        MD5 = 'md5'
+        SHA = 'sha'
+        SHA256 = 'sha256'
+        SHA512 = 'sha512'
+        CRC = 'crc'
```

Comment:
CRC isn't one thing, and none of them are useful anyway

```suggestion
        S3_ETAG = S3 ETag'
```

Reply:
Haha I stole crc from indexd bc its one of their options for hashes